packages/context/src/context.ts

```diff
@@ -150,6 +248,19 @@ export class Context {
    * - return `true` to include the binding in the results
    * - return `false` to exclude it.
    */
+  find<ValueType = BoundValue>(
+    pattern?: string | RegExp,
+  ): Readonly<Binding<ValueType>>[];
+
+  /**
+   * Find bindings using a filter function
+   * @param filter A function to test on the binding. It returns `true` to
+   * include the binding or `false` to exclude the binding.
+   */
+  find<ValueType = BoundValue>(
+    filter: BindingFilter,
+  ): Readonly<Binding<ValueType>>[];
```

Comment:
@raymondfeng I believe these two overloads are not needed. VSCode has suggested to combine them into a single method signature, which we have already in `master`:

```ts
find<ValueType = BoundValue>(
  pattern?: string | RegExp | BindingFilter,
): Readonly<Binding<ValueType>>[] {
```

Can you revert these changes in `find` declaration please?